plugin/priv_cc.lua

```diff
@@ -0,0 +1,8 @@
+-- if you need to set a special name for privilege, use "beerchat.name_priv_cc = spec_name" in minetest.conf
+local name_priv2cc = minetest.settings:get("beerchat.priv_cc.name")
+if name_priv2cc == '' then name_priv2cc = 'chat_creator' end
+minetest.log("info", "to create channel add new priv: " .. name_priv2cc)
+minetest.register_privilege(name_priv2cc, {
+        description = "Allows you to create a channel (mod beerchat)",
+        give_to_singleplayer = false
+})
```

Comment:
im thinking that it might be worth checking for the priv existing so that people can use one that already exists? would need to wrap the registration + check in a on mods loaded callback so that mod loading isnt an issue

Reply:
example check:
```lua
-- Privilege registration (if needed)
core.register_on_mods_loaded(function()
	if not core.registered_privileges[smartshop.report_priv] then
		core.register_privilege(smartshop.report_priv, {
			description = S("Allow the use of smreport command."),
			give_to_singleplayer = false
		})
	end
end)
```